src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerSyncService.kt

```diff
@@ -51,7 +52,8 @@ class PrisonerSyncService(
 
   private fun syncByNomsNumber(offenderIdDisplay: String) {
     nomisService.getOffender(offenderIdDisplay)?.run {
-      prisonerIndexService.sync(this)
+      val incentiveLevel = this.bookingId?.let { bookingId -> incentivesService.getCurrentIncentive(bookingId) }
+      prisonerIndexService.sync(this, incentiveLevel)
     }
```

Comment:
does the sync take place if bookingId isn't present?

Reply:
overall sync does but there just won't be an IEP